Test/CaptureFileReaderTest.cs

```diff
@@ -18,7 +18,7 @@ public class CaptureFileReaderDeviceTest
         [Test]
         public void CaptureInfinite()
         {
-            var device = new CaptureFileReaderDevice("../../capture_files/ipv6_http.pcap");
+            var device = new CaptureFileReaderDevice("../../../capture_files/ipv6_http.pcap");
```

Comment:
Can you update the commit body to mention why these paths have to be adjusted?

Reply:
I will create a separate pull request for that. Using relative paths in this way (`../..` or `../../..`) does not work quite well, when shallow copying (ReSharper/Rider for instance) is used. There should be some further changes required.

Reply:
Added `TestHelper` to resolve file names, 
https://github.com/chmorgan/sharppcap/pull/61/commits/f18ab0089f25e9e0a516b657cd787d7bce3a7fe5

It works even if shallow copying is enabled.